### ISSUE TITLE
Error thrown with Vue 3 - Cannot Find vue.esm.js

### ISSUE THREAD

**Issue by ryanbenson** [vue-styleguidist/vue-styleguidist#989]
**Current behavior**

An error is thrown when running `npm run styleguide` in Vue 3, when using `vue cli` to create a new project and add styleguidist. Error: `Error: Cannot find module 'vue/dist/vue.esm.js'`

After looking into `node_modules/vue/dist` there's no file with that name there, but there's:

* `vue.esm-bundler.js`
* `vue.esm-browser.js`

**To reproduce**

Sample repo: https://github.com/ryanbenson/vue3-styleguidist

Steps I took to make this:

Using Node 12.18.4 on Mac

```console
vue create hello-world
cd hello-world
vue add styleguidist
npm run styleguide
```

Minor note, it would error with an issue with tabs vs spaces first with the files that are added with `vue add styleguidist`, after fixing the tabs, and re-running it again, it produces the error.

While creating the Vue 3 project:

* Manual configuration with: Choose vue version (Vue 3), babel, Router, Vuex, CSS (node-sass), Linter, unit, e2e

**Expected behavior**

When running `npm run styleguide`, it shouldn't throw an error, but successfully build the style guide.

**Comment by SmallComfort** [comment]
Hello, when does this bug fix publish?

**Comment by elevatebart** [comment]
This bug is already published and was not enough to make vue-styleguidist compatible with vue3.
To follow the progress of vue 3 compatibility, have a look at #997 and #1002.
